Rocket.Chat/Controllers/Preferences/ChangeAppIcon/ChangeAppIconViewController.swift

```diff
@@ -61,7 +61,8 @@ extension ChangeAppIconViewController: UICollectionViewDataSource {
             fatalError("Could not dequeue reuable cell as ChangeAppIconCell")
         }
 
-        let iconName = viewModel.availableIcons[indexPath.row]
+        let iconName = Array(viewModel.availableIcons.keys)[indexPath.row]
+        let iconAccessibilityName = Array(viewModel.availableIcons.values)[indexPath.row]
```

Comment:
Maybe you could just get the value out by using the iconName as the index. Should be safer down the line. 

```swift
viewModel.availableIcons[iconName]
```